keras_cv/ops/box_matcher_test.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.ops.box_matcher import ArgmaxBoxMatcher
+
+
+class ArgmaxBoxMatcherTest(tf.test.TestCase):
+    def test_box_matcher_invalid_length(self):
+        fg_threshold = 0.5
+        bg_thresh_hi = 0.2
+        bg_thresh_lo = 0.0
+
+        with self.assertRaises(ValueError):
```

Comment:
let's also assert the error message as well.

Reply:
Done.